lib/si-layer-cache/src/pg.rs

```diff
@@ -37,13 +39,14 @@ where
 
 impl<K> PgLayer<K>
 where
-    K: AsRef<[u8]> + Copy + Send + Sync,
+    K: AsRef<[u8]> + Copy + Send + Sync + std::hash::Hash + std::cmp::Eq,
```

Comment:
Did you end up needing these extra trait bounds? I don't expect we'll ever have a key that won't be hashable or support equality, but I think these were for the FromSql stuff?

Reply:
Both are required to insert into the hashmap on line 90 in `pg.rs`. Should I propagate the traits bounds to the other places we define k?

Reply:
You won't need them once our code merges, since K will always be an &str or String.